src/library/autodj/dlgautodj.h

```diff
@@ -23,13 +23,14 @@ class DlgAutoDJ : public QWidget, public Ui::DlgAutoDJ, public LibraryView {
               Library* pLibrary,
               AutoDJProcessor* pProcessor, TrackCollection* pTrackCollection,
               KeyboardEventFilter* pKeyboard);
-    virtual ~DlgAutoDJ();
-
-    void onShow();
-    void onSearch(const QString& text);
-    void loadSelectedTrack();
-    void loadSelectedTrackToGroup(QString group, bool play);
-    void moveSelection(int delta);
+    virtual ~DlgAutoDJ() override;
```

Comment:
I know we don't "need" it, but I like to always use `virtual xyz() override;` to make it 100% explicit. Apparently "some compilers" don't like it when you don't include it.

http://stackoverflow.com/questions/4895294/c-virtual-keyword-for-functions-in-derived-classes-is-it-necessary

I'd prefer to keep it there unless you strongly want me to remove it